<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="zentere-Latitude-E6420" name="com.hs.gofmpro.testcase.TestCreateSpaceCategory" tests="1" failures="1" timestamp="25 Apr 2019 06:04:42 GMT" time="0.000" errors="0">
  <testcase name="loginTest" time="0.000" classname="com.hs.gofmpro.testcase.TestCreateSpaceCategory">
    <failure type="org.testng.TestNGException" message="
Method public void com.hs.gofmpro.testcase.TestCreateSpaceCategory.loginTest(java.lang.String) throws java.lang.InterruptedException,java.io.IOException requires a @DataProvider named : GofmInput">
      <![CDATA[org.testng.TestNGException: 
Method public void com.hs.gofmpro.testcase.TestCreateSpaceCategory.loginTest(java.lang.String) throws java.lang.InterruptedException,java.io.IOException requires a @DataProvider named : GofmInput
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:265)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- com.hs.gofmpro.testcase.TestCreateSpaceCategory -->
